microhapulator/data/template.html

```diff
@@ -86,6 +86,48 @@ <h2>Table of Contents</h2>
 
             <a name="readqc"></a>
             <h2>Read QA/QC</h2>
+            <h5>Ambiguous Reads</h5>
+            {% if reads_are_paired %}
+            <p>
+                If over {{"{:.0f}".format(ambiguous_read_threshold * 100)}}% of bases in a read are ambiguous ('N'), both reads in the pair are filtered out prior to read merging and excluded from all subsequent analysis.
+                The table below shows the number of read pairs that were removed in each sample and which read(s) in the pair contained too many ambiguous bases.
+            </p>
+            <table>
+                <tr>
+                    <th>Sample</th>
+                    <th class="alnrt">R1 Only</th>
+                    <th class="alnrt">R2 Only</th>
+                    <th class="alnrt">R1 and R2</th>
+                    <th class="alnrt">Pairs Removed</th>
```

Comment:
Testing this on the NimaGen data, this looks good. It's tough to put this in context without a denominator though. What is the number of total reads? What is the percentage of discarded ambiguous reads?

(We'd then need to change "Total Reads" below to something else. "Unambiguous reads" is a mouthful...maybe "filtered reads" instead?)

Reply:
Added a "Pairs Kept" column to provide context. "Total Reads" has also been changed to "Filtered Reads"